var/spack/repos/builtin/packages/plumed/package.py

```diff
@@ -57,6 +57,9 @@ class Plumed(AutotoolsPackage):
     variant('shared', default=True, description='Builds shared libraries')
     variant('mpi', default=True, description='Activates MPI support')
     variant('gsl', default=True, description='Activates GSL support')
+    variant('arrayfire', default='disable',
+            values=('disable', 'cpu', 'cuda', 'opencl'),
```

Comment:
Note that I am not sure about `disable` here, I think it should be `disabled` but maybe someone can come up with something better.

Reply:
We usually use the string `none` if there is no support.

Reply:
Yeah, not sure why I didn't like `none` at first... I've repushed.